tcms/testruns/models.py

```diff
@@ -232,6 +228,14 @@ def _get_absolute_url(self):
         # NOTE: this returns the URL to the TestRun containing this TestExecution!
         return reverse("testruns-get", args=[self.run_id])
 
+    @property
+    def actual_duration(self):
+        if self.stop_date is None or self.start_date is None:
+            return None
+        elif self.start_date is None and self.start_date is None:
+            return None
```

Comment:
This implementation is overzealous, try figuring out why. 

See the hints reported by CodeClimate and also by pylint (with less verbosity).